app/routes.py

```diff
@@ -1,2 +1,247 @@
-from flask import Blueprint
+from flask.wrappers import Response
+import requests
+import os
+from app import db
+from .models.task import Task
+from flask import Blueprint, make_response, request, jsonify
+from datetime import datetime
+from dotenv import load_dotenv
+from .models.goal import Goal
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+
+@tasks_bp.route("", methods=["POST", "GET"])
+def handle_tasks():     
+    if request.method == "POST":
+        request_body = request.get_json()
+        if "title" not in request_body or "description" not in request_body or "completed_at" not in request_body:
+            return {
+                "details": "Invalid data"
+            }, 400
+    
+        new_task = Task(title=request_body['title'], 
+                        description=request_body['description'],
+                        completed_at=request_body['completed_at'])
+        
+        db.session.add(new_task)
+        db.session.commit()
+        response_body = {
+            "task": new_task.resp_json()
+        }
+        return jsonify(response_body), 201
+
+    elif request.method == "GET":
+        sorting_tasks = request.args.get("sort")
+        
+        tasks = Task.query.all()
+        tasks_response = []
+
+        if sorting_tasks == "asc":
+            asc_order = Task.query.order_by(Task.title.asc())
+            new_order = []
+        
+            for task in asc_order:
+               new_order.append(task.resp_json())
+            return jsonify(new_order), 200
+
+        elif sorting_tasks == "desc":
+            desc_order = Task.query.order_by(Task.title.desc())
+            new_order = []
+        
+            for task in desc_order:
+               new_order.append(task.resp_json())
+            return jsonify(new_order), 200
+
+
+        for task in tasks:
+            tasks_response.append(task.resp_json())
+
+        return jsonify(tasks_response), 200
```

Comment:
Great job Juliana!! I see a lot of great refactoring opportunities here. 

💡 Refactoring hint: notice how you're creating `new_order` within each conditional and also iterating through a list? I'd love to go over how to refactor this particular section with you during our 1:1 😄

Reply:
More time means more refactoring! :) I agree, great opportunities for the entire project to be refactored, just not enough time.